### PR TITLE
Add glossary entries for remote read.

### PR DESCRIPTION
**Pull request by brian-brazil**
Update promdash entry.

@juliusv

### CODE REVIEW

File: content/docs/introduction/glossary.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ to email, Pagerduty, Slack etc.
 
 ### Promdash
 
-[Promdash](../../visualization/promdash/) is a Ruby-on-Rails dashboard builder for Prometheus.
-It is similar at a high-level to Grafana, but is specific to Prometheus.
+Promdash is a deprecated Ruby-on-Rails dashboard builder for Prometheus. It was
```
Comment (grobie) [inline content/docs/introduction/glossary.md]: That's not what we have in master. I think we should merge master into next-release first.

File: content/docs/introduction/glossary.md [hunk 2]
```diff
@@ -71,6 +71,25 @@ The [Pushgateway](../../instrumenting/pushing/) persists the most recent push
 of metrics from batch jobs. This allows Prometheus to scrape their metrics
 after they have terminated.
 
+### Remote Read
+
+Remote read is a Prometheus feature that allows transparent reading time series from
```
Comment (juliusv) [inline content/docs/introduction/glossary.md]: "transparent reading time series" -> "transparent reading of time series"

File: content/docs/introduction/glossary.md [hunk 3]
```diff
@@ -71,6 +71,25 @@ The [Pushgateway](../../instrumenting/pushing/) persists the most recent push
...
+### Remote Read Adapter
+
+Not all systems directly support remote read. A remote read adapter sits between
+Prometheus and another system, converting time series requests and responses between them.
+
+### Remote Read Endpoint
+
+A remote read endpoint is what Prometheus talks to when doing a remote read.
+
+### Remote Write
```
Comment (juliusv) [inline content/docs/introduction/glossary.md]: Aren't there also the same "Adapter" and "Endpoint" variants for "Remote Write"?

**Comment by brian-brazil** [comment]
Updated.

File: content/docs/introduction/glossary.md [hunk 4]
```diff
@@ -71,6 +71,35 @@ The [Pushgateway](../../instrumenting/pushing/) persists the most recent push
 of metrics from batch jobs. This allows Prometheus to scrape their metrics
 after they have terminated.
 
+### Remote Read
+
+Remote read is a Prometheus feature that allows transparent of reading time series from
```
Comment (juliusv) [inline content/docs/introduction/glossary.md]: "of reading" -> "reading of"

**Comment by juliusv** [comment]
:+1: after last nit

File: content/docs/introduction/glossary.md (hunk 4 above)
Comment (brian-brazil) [inline content/docs/introduction/glossary.md]: fixed

**Pull request merged**